mapof.go

```diff
@@ -245,14 +287,12 @@ func (m *MapOf[K, V]) doStore(key K, valueFn func() V, loadIfExists bool) (V, bo
 				// Create and append a new bucket.
 				newb := new(bucketPadded)
 				newb.keys[0] = unsafe.Pointer(&key)
-				value := valueFn()
```

Comment:
The `valueFn` call should be kept here. That's to avoid having it called multiple times in a single `Compute` call. This may happen when the table gets resized (L282-285).

Reply:
Ah, I see the goto now. Sorry about that.

Reply:
No problem at all. I've just released v2.1.0. Please let me know if you face any issues with the `Compute` method.